packages/origin-247-certificate/src/onchain-certificate/deployContracts.ts

```diff
@@ -2,18 +2,23 @@ import { Contracts } from '@energyweb/issuer';
 import { providers } from 'ethers';
 import { DeploymentProperties } from './types';
 
+type DeployParameters = {
```

Comment:
this could be imported from @energyweb/issuer

Reply:
implemented on [f4a209b](https://github.com/energywebfoundation/origin-247-sdk/pull/104/commits/f4a209bc63a412fb229c376e64d8a1496a6832c5)